customers/views.py

```diff
@@ -93,16 +66,8 @@ def update(customer_id):
             updated_customer.last_name = request_data["last_name"]
```

Comment:
how can you reduce this code by using the values from `request_date` more efficiently with your schema?